src/Tools/dotnet-monitor/Extensibility/ToolsExtensionRepository.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Logging;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.Extensibility
+{
+    internal class ToolsExtensionRepository : ExtensionRepository
```

Comment:
This is very similar to `FolderExtensionRepository` - these may be refactored later, but Justin recommended we keep them separate for the time being.